debian/nfd.init

```diff
@@ -0,0 +1,137 @@
+#! /bin/sh
```

Comment:
it is required by the debian policy: https://www.debian.org/doc/debian-policy/ch-opersys.html#s-writing-init